frontend/src/styles/material.sass

```diff
@@ -43,10 +43,28 @@ snack-bar-container
     & span
         width: 100%
 
-
 .mat-radio-button.mat-accent.mat-radio-checked .mat-radio-outer-circle
     border-color: $royal-blue
 
 .mat-radio-button.mat-accent .mat-radio-inner-circle
     color: $royal-blue
     background-color: $royal-blue
+
+.mat-slide-toggle-thumb
+    width: 10px !important
+    height: 10px !important
+    transform: translate(50%, 50%)
+
+.mat-slide-toggle-bar
+    background-color: $royal-blue
+    border-radius: 15px !important
+    height: 16px !important
+
+.mat-slide-toggle-thumb-container
+    top: -2px !important
+
+.mat-slide-toggle.mat-checked:not(.mat-disabled) .mat-slide-toggle-bar
+    background-color: $royal-blue
+
+.mat-slide-toggle.mat-checked:not(.mat-disabled) .mat-slide-toggle-thumb
+    background-color: $white
```

Comment:
Why do you define classes 2 times?

Reply:
I have been trying for 2 hours but did'nt find how this style can work in another implementation (

Reply:
Fix toggle disabled color (figma). I didn't find any way to not define classes 2 times for get this style